src/routes/safe/components/AddressBook/index.tsx

```diff
@@ -126,7 +126,7 @@ const AddressBookTable = (): ReactElement => {
     // close the modal
     setEditCreateEntryModalOpen(false)
     // update the store
-    dispatch(addressBookAddOrUpdate(makeAddressBookEntry(entry)))
+    dispatch(addressBookAddOrUpdate(makeAddressBookEntry({ ...entry, address: checksumAddress(entry.address) })))
```

Comment:
The checksum fn is now indeed not in the reducer but it can still throw.
It will crash the app if not caught here.

In practice it shouldn't happen because both import and creating an entry manually will have its own validation.

Reply:
indeed, I focused on the runtime data. My goal was to ensure that data that reaches `checksumAddress` was a valid address.